tests_integration/test_installation.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import subprocess
+from pathlib import Path
+
+from snowcli import __about__
+
+
+# @pytest.mark.integration
```

Comment:
I think we should mark this as integration or `e2e` WDYT?

Reply:
I think we should mark it as integration for now, we can create new mark when we will have more this kind of tests